internal/format/format.go

```diff
@@ -48,6 +49,11 @@ func New(cfg Config) (Format, error) {
 // move into go templates
 func (f *Format) Render() error {
 	modules := f.Config.GetSource()
+
+	if len(modules) == 0 {
+		return errors.New("no modules found")
```

Comment:
Add instruction on what users should be doing to unblock?

Reply:
makes sense, in fact, each module is handling its own way to unblock users. I remove this if statement because it isn't needed